Sources/NostrSDK/Events/Tags/IdentifierTagInterpreting.swift

```diff
@@ -1,29 +0,0 @@
-//
-//  IdentifierTagInterpreting.swift
-//  
-//
-//  Created by Terry Yiu on 12/16/23.
-//
-
-import Foundation
-
-public protocol IdentifierTagInterpreting: NostrEvent {}
-public extension IdentifierTagInterpreting {
-    /// The identifier of the event. For parameterized replaceable events, this identifier remains stable across replacements.
-    /// This identifier is represented by the "d" tag, which is distinctly different from the `id` field on ``NostrEvent``.
-    var identifier: String? {
-        firstValueForTagName(.identifier)
-    }
-
-    /// The event coordinates that can be used to fetch this replaceable event's kind, pubkey, and identifier from a relay.
-    ///
-    /// - Parameters:
-    ///   - relayURL: A relay URL that this replaceable event could be found.
-    func identifierEventCoordinates(_ relayURL: URL? = nil) -> EventCoordinates? {
```

Comment:
Non-parameterized replaceable events do not have identifiers but they do have event coordinates, so the way this protocol was structured didn't make sense.